packet_forwarder/src/lora_pkt_fwd.c

```diff
@@ -1156,7 +1156,7 @@ static int send_tx_ack(uint8_t token_h, uint8_t token_l, enum jit_error_e error,
             break;
         case JIT_ERROR_OK:
             memcpy((void *)(buff_ack + buff_index), (void *)"\"NONE\"", 6);
-            buff_index += 9;
+            buff_index += 6;
```

Comment:
There are several ways make this code adr cleaning before this suggestion, but we could use memfd to cleanup memory management. But it may not work on BsD. 

TBC: not actually suggestion but ICYDK

Reply:
I’m on phone and fully recognize my previous comment is intelligible